src/hooks/api/events.ts

```diff
@@ -251,13 +317,34 @@ const useDeleteEventByIdMutation = (configuration = {}) =>
     ...configuration,
   });
 
-const getEventsByCreator = async ({ headers, ...queryData }) => {
+const getEventsByCreator = async ({
+  headers,
+  q,
+  disableDefaultFilters,
+  embed,
+  limit,
+  start,
+  workflowStatus,
+}: {
+  headers: Headers;
+  q: string;
+  disableDefaultFilters: string;
+  embed: string;
+  limit: string;
+  start: string;
+  workflowStatus: string;
+}) => {
```

Comment:
I sometimes wonder if it's not clearer to make an intermediate type? And use it in the function? 

```
type GetEventsByCreatorArguments= {
  headers: Headers;
  q: string;
  disableDefaultFilters: string;
  embed: string;
  limit: string;
  start: string;
  workflowStatus: string;
}
```